### PR TITLE
Translate Colaboration Agreement.md

### PR DESCRIPTION
**Pull request by Bokaiii**

### CODE REVIEW

File: project_proposal/gruppekontrakt.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+| Simon Jørgensen               | sijo819@student.sdu.dk | 42 83 25 60 |
+| Kenneth Munk Christiansen     | Kechr19@student.sdu.dk | 28 67 66 78 |
+
+## Expectations and goals
+- Each group member has a workload of 10 hours, as well as 2 hours for meetings.
+- The outcome of the project must comply with the minimum requirements. Further
+work will be prioritised according to the MoSCoW model (must, should, could and
+won't have), since the workload is expected to be maintained.
+
+- Der forventes at gruppens medlemmer har en arbejdsmængde på ca. 10 timer,
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+- The group will meet online every friday at 8:00 am
+- If a member of the group is sick or is otherwise unable to attend and is unable to
+contact the remaining group members, the member must contact and inform the rest of the group
+as soon as possible.
+- For planned SCRUM meetings, an estimated workload of 10 hours/week will be distributed
+to each group member, and it is each members own responsibility to comply with the due date.
+- If the above-mentioned work is to be done in groups, worktimes are agreed upon in the group
+
+
+- Kører med det akademiske kvarter
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+```
+
+## Group Meetings
+- Meetings will occur every Friday, from 8:00 am on Discord
+- If a member is late or is absent, the remaining members can give a warning, and
+later talk to instructor.
+- A member must warn the remaining group members of planned absence as soon as possible
+
+
+- Møder foregår fast hver fredag, med start fra kl. 8.00 via Discord.
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+- Der forventes at gruppens medlemmer har en arbejdsmængde på ca. 10 timer,
+ekslusivt de estimerede 2 timer der sættes af til møder.
+- Resultatet af projektet skal overholde minimumskravene, efter dette er opnået
+prioriteres ekstra arbejde efter MoSCoW modellen (fra Must Have til Won't Have),
+da selvom minimumskravene er opnået, forventes det stadig at arbejdsmængden
+bibeholdes.
+
+## Work times
+- Meeting times follow the quarter system (15 minutes between defined time to actual start time)
+- The group will meet online every friday at 8:00 am
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Friday should be capitalized

Comment (nymann) [reply project_proposal/gruppekontrakt.md]: I have not used am/pm in my group contract document, should we include this?

File: project_proposal/gruppekontrakt.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+- If a member of the group is sick or is otherwise unable to attend and is unable to
+contact the remaining group members, the member must contact and inform the rest of the group
+as soon as possible.
+- For planned SCRUM meetings, an estimated workload of 10 hours/week will be distributed
+to each group member, and it is each members own responsibility to comply with the due date.
+- If the above-mentioned work is to be done in groups, worktimes are agreed upon in the group
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: is worktimes a real word?

File: project_proposal/gruppekontrakt.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+intern advarsel og efterfølgende tage en snak med vejleder.
+- Ved planlagt udblivelse, varsles dette til de andre gruppe medlemmer i så god
+tid som muligt.
+
+###  Organization of meetings
+```
+TODO; Ordstyrer bliver valgt udfra om mødet foregår online eller fysisk.
+Ved fysisk møde, skal ordstyren være en af de fremmødte.
+```
+- Kepet19 is responsible for writing summaries of meetings with jakra19 as alternate.
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Let's capitalize all usernames `jakra19` => `Jakra19`

File: project_proposal/gruppekontrakt.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+- All deadlines and internal agreements must be kept, and if not possible, logged.
+- Meeting agenda and eventual problems must be logged.
+
+
+- Referent bliver kepet19, og suppliant jakra19.
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+- Det forventes af hver gruppemedlem har læst op og fået styr på de emmer, vi
+arbejder med, inden møderne og arbejdsdagene.
+
+## Use and revision of the Colaboration Agreement
+- The Colaboration Agreement will be used during development
+- The Colaboration Agreement will be used during conflicts
+- The Colaboration Agreement can be revised, if need be
+- The new Colaboration Agreement must be approved by each member
+
+- Samarbejdsaftalen tages i brug løbende.
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+
+## Tools
+
+##### Primary Communication
+The primary means of communication for group meetings will be discord. 
+Written communication with instructor will take place over e-mail
+
+
+Discord bruges til interne gruppemøder, samt tekst beskeder. Skriftlig
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+
+ZenHub bruges i projektet (en browser extension til GitHub) til projektstyring.
+
+##### Implementation
+Git, via Github, will be used for project implementations, including report
+writing and code implementation. Logbook and summaries of meeting is found as
+wiki-pages in [management projektet](https://github.com/bierproductie/management).
+
+
+Git via GitHub bruges til implementering af projektet, herunder rapport
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+
+Discord bruges til interne gruppemøder, samt tekst beskeder. Skriftlig
+kommunikation hvor vejleder er involveret foregår over e-mail.
+
+
+##### Project Management
+For managing the project, zenhub (browser extension for Github) will be used.
+
+
+ZenHub bruges i projektet (en browser extension til GitHub) til projektstyring.
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+skrivning, kode implementering. Logbog og referater vil være at finde som wiki
+sider i [management projektet](https://github.com/bierproductie/management).
+
+##### Filesharing
+Filesharing will be used as required. Project related PDP documents is saved
+via Git-LFS in [management projektet](https://github.com/bierproductie/management).
+Files can also be shared via own server with the command: `scp`.
+
+
+Fildeling foregår efter behov. Projekt relaterede PDF dokumenter, gemmes via
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Delete this Danish section

File: project_proposal/gruppekontrakt.md [hunk 13]
```diff
@@ -0,0 +1,149 @@
+# Colaboration agreement
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Let's rename this to Group contract, to be consistent with the other document

File: project_proposal/gruppekontrakt.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+overholde dette, og der stilles derfor ikke krav om at arbejdet skal udføres om
+fredagen.
+    - Hvis ovenstående arbejde foregår i en gruppe, planlægges arbejdstiden i
+    gruppen.
+```
+TODO; indsæt vilkår ang. vejledermøde
+```
+
+## Group Meetings
+- Meetings will occur every Friday, from 8:00 am on Discord
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Should we exclude am/pm like suggested in the other comment?

File: project_proposal/gruppekontrakt.md [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+## Course participation
+- Every group member must have an understanding of the subjects, the group is
+working with, before meetings and work days.
+
+
+- Det forventes af hver gruppemedlem har læst op og fået styr på de emmer, vi
+arbejder med, inden møderne og arbejdsdagene.
+
+## Use and revision of the Colaboration Agreement
+- The Colaboration Agreement will be used during development
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: Let's rename this to Group contract, to be consistent with the other document

File: project_proposal/gruppekontrakt.md [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+The primary means of communication for group meetings will be discord. 
+Written communication with instructor will take place over e-mail
+
+
+Discord bruges til interne gruppemøder, samt tekst beskeder. Skriftlig
+kommunikation hvor vejleder er involveret foregår over e-mail.
+
+
+##### Project Management
+For managing the project, zenhub (browser extension for Github) will be used.
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: zenhub => ZenHub
Github => GitHub

File: project_proposal/gruppekontrakt.md [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+wiki-pages in [management projektet](https://github.com/bierproductie/management).
+
+
+Git via GitHub bruges til implementering af projektet, herunder rapport
+skrivning, kode implementering. Logbog og referater vil være at finde som wiki
+sider i [management projektet](https://github.com/bierproductie/management).
+
+##### Filesharing
+Filesharing will be used as required. Project related PDP documents is saved
+via Git-LFS in [management projektet](https://github.com/bierproductie/management).
```
Comment (nymann) [inline project_proposal/gruppekontrakt.md]: management projektet => management repository

**Review by nymann — CHANGES_REQUESTED** [review]

File: project_proposal/gruppekontrakt.md (hunk 5 above)
Comment (Bokaiii) [reply project_proposal/gruppekontrakt.md]: "Work times" er c:

**Pull request merged**